extensions/lifecycle/tasks/LifecycleColdStatusArchiveTask.js

```diff
@@ -36,7 +37,7 @@ class LifecycleColdStatusArchiveTask extends LifecycleUpdateTransitionTask {
               .setAttribute('target.key', key)
               .setAttribute('target.version', version)
               .setAttribute('target.accountId', accountId)
-              .setAttribute('target.owner', owner);
+              .setAttribute('target.ownerId', ownerId);
```

Comment:
this is a partial fix only.

Looking at backbeat (lifecycle) overall, there are _lots_ places where we set `target.owner` field : so either we should change them all, or change GC to extract the `owner` field from message (one liner)